Sources/Controllers/2.0/OAQuickSearchViewController.mm

```diff
@@ -1824,13 +1826,25 @@ - (void) reloadAddressData
 
 #pragma mark - OACategoryTableDelegate
 
-- (void) createPOIUIFIlter
+- (void)createPOIUIFilter
 {
     OAPOIUIFilter *filter = [[OAPOIFiltersHelper sharedInstance] getCustomPOIFilter];
     [filter clearFilter];
     OACustomPOIViewController *customPOI = [[OACustomPOIViewController alloc] initWithFilter:filter];
     customPOI.delegate = self;
-    [self.navigationController pushViewController:customPOI animated:YES];    
+    [self.navigationController pushViewController:customPOI animated:YES];
+}
+
+- (void)showRemoveFiltersView:(NSArray<OAPOIUIFilter *> *)filters
```

Comment:
You do not show a view (a UIView is just an element of a view controller like a button or a tableview) in this method, so this name is confusing. Please rename it to something like: ```showRemoveFiltersScreen```